src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -275,8 +237,6 @@ function PaymentMethodList({
 
     /**
      * Render placeholder when there are no payments methods
-     *
-     * @return {React.Component}
      */
```

Comment:
make description one liner

Reply:
I'm not sure I follow what the request here is